sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -232,30 +232,41 @@ case class RLike(left: Expression, right: Expression) extends StringRegexExpress
  * Splits str around pat (pattern is a regular expression).
  */
 @ExpressionDescription(
-  usage = "_FUNC_(str, regex) - Splits `str` around occurrences that match `regex`.",
+  usage = "_FUNC_(str, regex, limit) - Splits `str` around occurrences that match `regex`." +
+    "The `limit` parameter controls the number of times the pattern is applied and " +
```

Comment:
can we be more concise? e.g. presto's doc is just

"Splits string on delimiter and returns an array of size at most limit. The last element in the array always contain everything left in the string. limit must be a positive number."

Reply:
you should say if limit is ignored if it is a non-positive number.

Reply:
@rxin the underlying implementation of this method is Java.lang.String, correct? This method does allow non-positive values for `limit`, not sure what Presto is using. The text I've put here corresponds with the definition (rather long) from Java.lang.String.